### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by seannoaa**
Untested, but should be functional based on all pre-requisites having been installed.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+FROM ubuntu:22.04
+
+# docker build --platform=linux/amd64 -t labelmaker:latest .
+# docker run --platform=linux/amd64 -it labelmaker:latest /bin/bash
+
+# Install core developer dependencies
+RUN apt-get -y -m update && apt-get install -y build-essential wget autoconf git unzip python3 python3-pip texlive-base
```
Comment (lukenoaa) [inline Dockerfile]: I was able to build and run the container, but it appears that `lualatex` and other tex binaries aren't installed:

```bash
root@de056a77a901:/labelmaker# lualatex
bash: lualatex: command not found
root@de056a77a901:/labelmaker# latex
bash: latex: command not found
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ENV PATH="/miniconda/bin:${PATH}"
+
+# Install Mamba from Conda-Forge
+RUN conda install pandas click python=3
+
+# Install qrcode
+RUN pip install qrcode[pil]
+
+# Install Labelmaker
+RUN git clone https://github.com/aomlomics/labelmaker
```
Comment (lukenoaa) [inline Dockerfile:24]: Python code works great.

**Review by lukenoaa — COMMENTED** [review]
Only thing that I couldn't get working was `lualatex`. Any thoughts on that?

**Comment by seannoaa** [comment]
@lukenoaa I cancelled the PR and re-opened because I wasn't sure if that was required to see the new Dockerfile changes. Hopefully you are looking at a new PR with lines 27 and 28 indicating the new lualatex install.

File: Dockerfile (hunk 1 above)
Comment (seannoaa) [reply Dockerfile]: Fixed in new PR. I've tested and all seems fine running the following test command from inside the /labelmaker directory: 

./generate_labels.py \
  --project ProjectX \
  --contact MySurname \
  --date 180929 \
  --sample_list sample_list.txt \
  --num_replicates 1 \
  --separator '_'

**Comment by lukenoaa** [comment]
Thanks Sean! Testing now. Good news is that `lualatex` is installed and runs. Bad news is that some of the style files (eg, fullpage.sty) that come with Mac TeX or TeX Live are not found. These are: fullpage.sty, pdftexcmds.sty, infwarerr.sty, pdftexcmds.sty. I tested and running the following command fixes everything. Can you add this to the Dockerfile?:

```
apt-get install texlive-latex-extra
```

**Comment by lukenoaa** [comment]
Command:

`docker build --platform=linux/amd64 -t labelmaker:latest .`

is hanging on:

```
 => [7/8] RUN apt-get update && apt-get install -y     texlive-base     texlive-latex-extra     && apt-get clean     && rm -rf /var/lib/apt/lists/*                                                                                        152607.3s
 => => # questions will narrow this down by presenting a list of cities, representing
 => => # the time zones in which they are located.
 => => #   1. Africa      4. Australia     7. Atlantic Ocean  10. Pacific Ocean
 => => #   2. America     5. Arctic Ocean  8. Europe          11. US
 => => #   3. Antarctica  6. Asia          9. Indian Ocean    12. None of the above
 => => # Geographic area:
```

**Comment by lukenoaa** [comment]
Following @seannoaa's suggestion, added `ARG DEBIAN_FRONTEND=noninteractive` at start of Dockerfile, which tells Debian-based systems to run in non-interactive mode.

**Pull request merged**